addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -420,7 +420,12 @@ def _import_fattura_pa(self, tree, invoice):
                                 ('partner_id.id', '=', invoice_form.partner_id.commercial_partner_id.id)
                                 ])
                         else:
-                            bank = self.env['res.partner.bank'].search([('acc_number', '=', elements[0].text)])
+                            bank = self.env['res.partner.bank'].search([
+                                ('acc_number', '=', elements[0].text),
+                                ('company_id', '=', invoice_form.company_id.id),
```

Comment:
Don't you need an `_origin` here? I'm not sure if a form emulator is returning always New record or not

Reply:
The other two places in this method using the company_id are just getting the .id, so I did the same for this.